react-client/src/containers/AddReview/AddReview.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import style from './AddReview.module.scss'
+import { useState } from 'react'
+import { Link, useLocation } from 'react-router-dom'
+import { addNewReview } from '../../services/Podcasts'
+
+const AddReview = () => {
+  const location = useLocation()
+  const podcastId = location.pathname.replace('/podcast/new-review/', '')
+  const [ratingStars, setRatingStars] = useState(['\u2606', '\u2606', '\u2606', '\u2606', '\u2606'])
```

Comment:
the state of the rating should be a number and not a font. in the ui you should translate this number into a font